<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDCCTExpert</class>
 <widget class="QMainWindow" name="FormDCCTExpert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DCCT Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="titleGroupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>311</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>     Timing</string>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Delay</string>
     </property>
    </widget>
    <widget class="QENumericEdit" name="txtDelay">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setDelay</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="leadingZeros">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QELabel" name="lblDelay">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getDelay</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string/>
     </property>
    </widget>
    <widget class="QENumericEdit" name="txtWidth">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setWidth</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="leadingZeros">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QELabel" name="lblWidth">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getWidth</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
    <widget class="QESimpleShape" name="ledEnabled">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>0</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):isEnabled</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string/>
     </property>
    </widget>
    <widget class="QEPushButton" name="btnOFF">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):enable</string>
     </property>
     <property name="clickText">
      <string>0</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="btnON">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):enable</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="titleGroupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>660</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	background-color: rgb(255, 255, 255);
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>20</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="titleGroupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>311</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>DVM Settings</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Apperature</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblApp">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getNPLC</string>
     </property>
    </widget>
    <widget class="QEComboBox" name="cbApp">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>79</width>
       <height>20</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setNPLC</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Range</string>
     </property>
    </widget>
    <widget class="QEComboBox" name="cbDVMRange">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>79</width>
       <height>20</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setDvmRange</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblDVMRange">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getDvmRange</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="titleGroupBox_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>320</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>Trigger</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Trigger Source</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblSource">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getTriggerSource</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblSlope">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getTriggerSlope</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Trigger Slope</string>
     </property>
    </widget>
    <widget class="QEComboBox" name="cbSlope">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setTriggerSlope</string>
     </property>
    </widget>
    <widget class="QEComboBox" name="cbAuto">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setTriggerDelayAuto</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Trigger Auto</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblAuto">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getTriggerDelayAuto</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbSource">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>IMM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EXT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BUS</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QEComboBox" name="cbRange">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="variable" stdset="0">
     <string>$(group):setDcctRange</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelX">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Core Temperature</string>
    </property>
   </widget>
   <widget class="QELabel" name="lblTemp">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>$(group):getTemperature</string>
    </property>
   </widget>
   <widget class="QELabel" name="lblVoltage">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>$(group):getVoltage</string>
    </property>
    <property name="precision">
     <number>6</number>
    </property>
    <property name="useDbPrecision">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lifetime</string>
    </property>
   </widget>
   <widget class="QELabel" name="qelabel_14">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>SR-DI:getBeamLifetime</string>
    </property>
   </widget>
   <widget class="QELabel" name="lblCurrent">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>$(group):getDcctCurrent</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Energy</string>
    </property>
   </widget>
   <widget class="QELabel" name="qelabel_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>SR-DI:getBeamEnergy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Range</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltage</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupRange">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>332</x>
      <y>142</y>
      <width>331</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>DCCT-Range</string>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Range</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="btnOFF_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
     <property name="variable" stdset="0">
      <string>SR-DI-DCCT:test</string>
     </property>
     <property name="clickText">
      <string>0</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="btnON_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
     <property name="variable" stdset="0">
      <string>SR-DI-DCCT:test</string>
     </property>
    </widget>
    <widget class="QEComboBox" name="cbApp_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>79</width>
       <height>20</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>SR-DI-DCCT:setRange</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblDelay_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>SR-DI-DCCT:getStatus</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Functional Test</string>
     </property>
    </widget>
   </widget>
   <widget class="QChartView" name="dcctPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>660</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEComboBox</class>
   <extends>QComboBox</extends>
   <header>QEComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QENumericEdit</class>
   <extends>QWidget</extends>
   <header>QENumericEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QEPushButton</class>
   <extends>QPushButton</extends>
   <header>QEPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>QESimpleShape</class>
   <extends>QSimpleShape</extends>
   <header>QESimpleShape.h</header>
  </customwidget>
  <customwidget>
   <class>QSimpleShape</class>
   <extends>QWidget</extends>
   <header>QSimpleShape.h</header>
  </customwidget>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>QtCharts</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
